lib/blogit/models/post/meta.ex

```diff
@@ -120,12 +121,27 @@ defmodule Blogit.Models.Post.Meta do
   defp create_from_map(data, file_path, repository, %{raw: raw, name: name, language: language})
        when is_map(data) do
     path = Path.join(posts_folder(), file_path)
-    file_info = repository.provider.file_info(repository.repo, path)
-
-    created_at = data["created_at"] || file_info[:created_at]
-    updated_at = data["updated_at"] || file_info[:updated_at]
-    author = data["author"] || file_info[:author]
-    author = if author == "", do: "Anonymous", else: author
+    
+
+    created_at = data["created_at"]
+    updated_at = data["updated_at"]
+    author = data["author"]
+    {created_at2, updated_at2, author2} =
+      if created_at == nil do
+        IO.inspect "get from REPO using git log:" <> file_path
+        file_info = repository.provider.file_info(repository.repo, path)
+        created_at3 = file_info[:created_at] || "2020-01-26T01:01:01"
```

Comment:
Use `DateTime.utc_now() |> DateTime.to_iso8601()` for the default. Don't hardcode dates.
Also don't name them here, just return:
```
{
  file_info[:created_at] || DateTime.utc_now() |> DateTime.to_iso8601(),
  updated_at || file_info[:updated_at] || DateTime.utc_now() |> DateTime.to_iso8601(),
  author || file_info[:author] || "Anonymous"
}
```